src/RangeEnclosures.jl

```diff
@@ -1,30 +1,33 @@
 module RangeEnclosures
 
-using Requires, Reexport
+using DynamicPolynomials, Requires, Reexport
 @reexport using IntervalArithmetic
 const Interval_or_IntervalBox = Union{Interval, IntervalBox}
 
 #=================
 Available methods
 ==================#
 const available_solvers = [:IntervalArithmetic,
-                           :IntervalOptimisation,
                            :TaylorModels,
-                           :SumOfSquares,
                            :BranchandBound]
 
-const optional_solvers = [:AffineArithmetic]
+const optional_solvers = [:AffineArithmetic, :IntervalOptimisation, :SumOfSquares]
 
 include("intervals.jl")
 include("taylormodels.jl")
-include("sdp.jl")
```

Comment:
Why did you remove this include?

Reply:
it depends on SumOfSquares.jl which is an optional dependency, it's imported in the init function